scrapy/extensions/memusage.py

```diff
@@ -36,6 +36,7 @@ def __init__(self, crawler):
         self.limit = crawler.settings.getint('MEMUSAGE_LIMIT_MB')*1024*1024
         self.warning = crawler.settings.getint('MEMUSAGE_WARNING_MB')*1024*1024
         self.report = crawler.settings.getbool('MEMUSAGE_REPORT')
+        self.check_interval = crawler.settings.getfloat('MEMUSAGE_CHECK_INTERVAL_SECONDS', 60.0)
```

Comment:
I think it is better to put a default value (60.0) to https://github.com/scrapy/scrapy/blob/master/scrapy/settings/default_settings.py.